packages/react-components/src/lib/utils/map.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
I think you should not use magic numbers and export them to `consts`

Reply:
Basically you are right, but not in this case. The calculation is good documented, so extracting to const doesn't add more info.
It's very specific